main/webapp/modules/core/scripts/project/operation-icons.js

```diff
@@ -0,0 +1,13 @@
+
+var OperationIconRegistry = {
+  map: new Map()
+};
+
+OperationIconRegistry.setIcon = function(operationId, iconPath) {
```

Comment:
```suggestion
OperationIconRegistry.setIcon = function(operationId, iconPath) {
  if (!iconPath.endsWith(".svg")) {
    throw new Error('Operation icon must be a path to a SVG file.');
  }
```

Just so that in case we want to move to a model in which the SVG markup is injected into the DOM we don't need a breaking change.